packages/core/src/lib/database/migrations/1746017213155-AlterTimeSlotMinuteEntityTable.ts

```
@@ -0,0 +1,105 @@
+import { DatabaseTypeEnum } from '@gauzy/config';
+
+export class AlterTimeSlotMinuteEntityTable1746017213155 implements MigrationInterface {
+	name = 'AlterTimeSlotMinuteEntityTable1746017213155';
+
+	/**
+	 * Up Migration
+	 *
+	 * @param queryRunner
+	 */
+	public async up(queryRunner: QueryRunner): Promise<void> {
+		console.log(chalk.yellow(this.name + ' start running!'));
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)